xarray_beam/__init__.py

```diff
@@ -17,7 +17,9 @@
 from xarray_beam._src.core import (
     Key,
     DatasetToChunks,
+    ValidateEachChunk,
     offsets_to_slices,
+    validate_chunk
```

Comment:
Could put these in `api.md`, too, so they show up in the docs.

Reply:
sgtm